test/powershell/Modules/Microsoft.PowerShell.Utility/string.tests.ps1

```diff
@@ -24,6 +29,14 @@
             Pop-Location
         }
 
+        It "Select-String does not throw on subdirectory (path without wildcard)" {
+            { select-string -Path  $pathWithoutWildcard "noExists" } | Should Not Throw
```

Comment:
Should not throw might not be a good test here because it doesn't throw with the current behavior -- a non-terminating error is written out.

Reply:
I think this test is fine but you should also verify the reported error (ProcessingFile,Microsoft.PowerShell.Commands.SelectStringCommand) either in the test or in another test.

Reply:
I skipped '-ErrorAction Stop' - Fixed.